Services/Tracking/classes/class.ilLPTableBaseGUI.php

```diff
@@ -47,7 +62,7 @@ public function __construct(
         // country names
         $this->lng->loadLanguageModule("meta");
         $this->anonymized = !ilObjUserTracking::_enabledUserRelatedData();
-        if (!$this->anonymized && $this->obj_id) {
+        if (!$this->anonymized && isset($this->obj_id)) {
```

Comment:
This is a change in the behaviour. Please check additionally whether obj_id > 0.